backend/src/routes/managerRoute.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import * as ManagerController from '../controllers/managerController';
+import { authenticateToken } from '../middleware/tokens';
+
+
+const router = express.Router();
+
+router.post('/', authenticateToken, ManagerController.create);
+router.get('/', authenticateToken, ManagerController.getAll);
```

Comment:
```suggestion
router.get('/', ManagerController.getAll);
```
I think getting all of the managers needs to be unauthenticated, so the front page can be viewable without an account